client/INLClient.py

```diff
@@ -55,7 +55,7 @@ def check_server(self, server):
             if self.get_client_info('manage_build_root'):
                 self.create_build_root()
 
-            load_modules = settings.CONFIG_MODULES[claimed['config']]
+            load_modules = self.get_config_modules()[claimed['config']]
```

Comment:
What happens if 'config' is not in `claimed` and whatever is not in `get_config_modules`...?